GeneralStateTests/stEWASMTests/validateBytecodeMissingVersionOneByteFromTxIsInvalid.json

```diff
@@ -0,0 +1,56 @@
+{
+    "validateBytecodeMissingVersionOneByteFromTxIsInvalid" : {
```

Comment:
Where did this came from?

Reply:
There existed a Filler for this test case but the filled json file was not pushed with it. So when I refilled all tests for the `call-checks` branch it got filled then I also include it.